pkg/frontend/query_result.go

```diff
@@ -597,15 +597,11 @@ func doDumpQueryResult(ctx context.Context, ses *Session, eParam *tree.ExportPar
 		mrs:        mrs,
 	}
 	//prepare output queue
-	exportParam.OutTofile = true
 	//prepare export param
 	exportParam.DefaultBufSize = getGlobalPu().SV.ExportDataDefaultFlushSize
-	exportParam.UseFileService = true
 	exportParam.FileService = getGlobalPu().FileService
 	exportParam.Ctx = ctx
 	defer func() {
-		exportParam.LineBuffer = nil
```

Comment:
these two buffers for resuing memory.  Is it really necessary to remove them?

Reply:
LineBuffer should be a local variable inside a function.  Not need to be a member variable.
LineStr is a also local variable.